### PR TITLE
Hotfix - Event description to take HTML

### PR DESCRIPTION
**Pull request by martypowell**
Some descriptions take HTML, this change allows this to happen.

### CODE REVIEW

**Comment by martypowell** [comment]
Resolves #248

File: js/apps/events-page/partials/eventDetails.html [hunk 1]
```diff
@@ -38,7 +38,7 @@ <h1>{{eventDetailsPage.data.Title}}</h1>
 		</div>	
 
 		<div ng-hide="eventDetailsPage.isLoading && !eventsPage.isError">
-			<p ng-if="eventDetailsPage.data.Description">{{eventDetailsPage.data.Description}}</p>
+			<div style="margin-bottom: 30px;" ng-if="eventDetailsPage.data.Description" ng-bind-html="eventDetailsPage.data.Description"></div>
```
[the remark this answers is not in the document]
Comment (mxsnyder) [reply js/apps/events-page/partials/eventDetails.html]: Their HTML will come through encoded anyway, so it's not a big deal to use a `<p>`.

[the remark this answers is not in the document]
Comment (martypowell) [reply js/apps/events-page/partials/eventDetails.html]: still think this is problematic, but changed.

**Comment by martypowell** [comment]
@mxsnyder please take another look, i realized eventRegistration had the same issue.

**Pull request merged**